From 222c490836a05d9beab2dec2e86b5027ffbc4b0e Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Mon, 13 Jul 2015 11:49:11 +0800
Subject: [PATCH] Visual Studio Builds: Adjust for All-in-one Builds

Adjust GlibEtcInstallRoot so that the ATK projects will integrate better
with the all-in-one solution file to build the GTK+ stack.
---
 build/win32/vs10/atk-version-paths.props  | 2 +-
 build/win32/vs11/atk-version-paths.props  | 2 +-
 build/win32/vs12/atk-version-paths.props  | 2 +-
 build/win32/vs9/atk-version-paths.vsprops | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/build/win32/vs10/atk-version-paths.props b/build/win32/vs10/atk-version-paths.props
index 4298972..6dfb2f5 100644
--- a/build/win32/vs10/atk-version-paths.props
+++ b/build/win32/vs10/atk-version-paths.props
@@ -3,7 +3,7 @@
   <PropertyGroup Label="UserMacros">
     <ApiVersion>1.0</ApiVersion>
     <VSVer>10</VSVer>
-    <GlibEtcInstallRoot>$(SolutionDir)\..\..\..\..\vs$(VSVer)\$(Platform)</GlibEtcInstallRoot>
+    <GlibEtcInstallRoot>$(SolutionDir)\..\..\vs$(VSVer)\$(Configuration)\$(Platform)</GlibEtcInstallRoot>
     <CopyDir>$(GLibEtcInstallRoot)</CopyDir>
     <DefDir>$(SolutionDir)$(Configuration)\$(PlatformName)\obj\$(ProjectName)\</DefDir>
     <AtkLibtoolCompatibleDllPrefix>lib</AtkLibtoolCompatibleDllPrefix>
diff --git a/build/win32/vs11/atk-version-paths.props b/build/win32/vs11/atk-version-paths.props
index bd9716f..b7648d3 100644
--- a/build/win32/vs11/atk-version-paths.props
+++ b/build/win32/vs11/atk-version-paths.props
@@ -3,7 +3,7 @@
   <PropertyGroup Label="UserMacros">
     <ApiVersion>1.0</ApiVersion>
     <VSVer>11</VSVer>
-    <GlibEtcInstallRoot>$(SolutionDir)\..\..\..\..\vs$(VSVer)\$(Platform)</GlibEtcInstallRoot>
+    <GlibEtcInstallRoot>$(SolutionDir)\..\..\vs$(VSVer)\$(Configuration)\$(Platform)</GlibEtcInstallRoot>
     <CopyDir>$(GLibEtcInstallRoot)</CopyDir>
     <DefDir>$(SolutionDir)$(Configuration)\$(PlatformName)\obj\$(ProjectName)\</DefDir>
     <AtkLibtoolCompatibleDllPrefix>lib</AtkLibtoolCompatibleDllPrefix>
diff --git a/build/win32/vs12/atk-version-paths.props b/build/win32/vs12/atk-version-paths.props
index d896155..e922295 100644
--- a/build/win32/vs12/atk-version-paths.props
+++ b/build/win32/vs12/atk-version-paths.props
@@ -3,7 +3,7 @@
   <PropertyGroup Label="UserMacros">
     <ApiVersion>1.0</ApiVersion>
     <VSVer>12</VSVer>
-    <GlibEtcInstallRoot>$(SolutionDir)\..\..\..\..\vs$(VSVer)\$(Platform)</GlibEtcInstallRoot>
+    <GlibEtcInstallRoot>$(SolutionDir)\..\..\vs$(VSVer)\$(Configuration)\$(Platform)</GlibEtcInstallRoot>
     <CopyDir>$(GLibEtcInstallRoot)</CopyDir>
     <DefDir>$(SolutionDir)$(Configuration)\$(PlatformName)\obj\$(ProjectName)\</DefDir>
     <AtkLibtoolCompatibleDllPrefix>lib</AtkLibtoolCompatibleDllPrefix>
diff --git a/build/win32/vs9/atk-version-paths.vsprops b/build/win32/vs9/atk-version-paths.vsprops
index 0feaded..5026595 100644
--- a/build/win32/vs9/atk-version-paths.vsprops
+++ b/build/win32/vs9/atk-version-paths.vsprops
@@ -10,7 +10,7 @@
 	/>
 	<UserMacro
 		Name="GLibEtcInstallRoot"
-		Value="$(SolutionDir)\..\..\..\..\vs$(VSVER)\$(PlatformName)"
+		Value="$(SolutionDir)\..\..\vs$(VSVER)\$(ConfigurationName)\$(PlatformName)"
 	/>
 	<UserMacro
 		Name="CopyDir"
-- 
2.3.5.windows.8

